docs/xaml/hot-reload.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "XAML Hot Reload for .NET MAUI"
+description: "Learn how to reload changes to your .NET MAUI XAML file instantly on your running app, so you don't have to rebuild your .NET MAUI project after every XAML change."
+ms.date: 06/23/2022
+---
+
+# XAML Hot Reload for .NET MAUI
+
+.NET Multi-platform App UI (.NET MAUI) XAML Hot Reload is a Visual Studio feature that enables you to view the result of XAML changes in your running app, without having to rebuild your project. Without XAML Hot Reload, you have to build and deploy your app every time you want to view the result of a XAML change.
+
+[!INCLUDE [docs under construction](~/includes/preview-note.md)]
+
+When your app is running, XAML Hot Reload parses your XAML edits and sends those changes to the running app. It preserves your UI state, since it doesn't recreate the UI for the full page, and updates changed properties on controls affected by edits. In addition, your navigate state and data will be maintained, enabling you to quickly iterate on your UI without losing your location in the app. Therefore, you'll spend less time rebuilding and deploying your apps to validate UI changes.
+
+By default, you don't need to save your XAML file to see the results of your edits. Instead, updates are applied immediately as you type. However, you can change this behavior to update only on file save. This can be accomplished by checking the **Apply XAML Hot Reload on document save** checkbox in the Hot Reload IDE settings available by selecting **Debug > Options> XAML Hot Reload** from the Visual Studio menu bar. Only updating on file save can sometimes be useful if you make bigger XAML updates and don't wish them to be displayed until they are complete.
+
+XAML Hot Reload supports simultaneous debugging of multiple platforms in Visual Studio. You can deploy an Android, iOS, and WinUI target at the same time to see your changes reflected on all three platforms at once. To debug on multiple platforms, see [How To: Set multiple startup projects](/visualstudio/ide/how-to-set-multiple-startup-projects).
```

Comment:
Is this true? I've not checked if it's even possible to launch a single project app on multiple platforms simultaneously.

Reply:
lol, I was just about to leave a comment about that. For Xamarin.Forms, it is true, since you can deploy multiple projects at the same time, and we will connect to all of them. For MAUI, _technically_ it does work if you have multiple separate projects set up, but you can't do it from a single project with multiple target frameworks.

IMO I would remove that part until we can actually add it for single project within Visual Studio since that's the intended use case for MAUI.